go-controller/pkg/informer/informer_test.go

```diff
@@ -30,7 +29,7 @@ func TestEventHandler(t *testing.T) {
 func newPod(name, namespace string) *kapi.Pod {
 	return &kapi.Pod{
 		Status: kapi.PodStatus{
-			Phase: v1.PodRunning,
+			Phase: kapi.PodRunning,
```

Comment:
This is not mentioned in the commit message.

Reply:
we were importing the same package twice with two different names, so i removed the second import, do you think i should add that to the message